### PR TITLE
Remove unused utility classes

### PR DESCRIPTION
**Pull request by cesidio**

### CODE REVIEW

File: app/frontend/src/styles/base/_utilities.scss [hunk 1]
```diff
@@ -183,10 +130,6 @@ input.button-as-link {
   }
 }
 
-.formatable-textarea {
-  min-height: 150px;
-}
-
 ul li + br,
```
Comment (alexbowen) [inline app/frontend/src/styles/base/_utilities.scss:133]: i think this was related to

Comment (cesidio) [reply app/frontend/src/styles/base/_utilities.scss:133]: related to?

Comment (csutter) [reply app/frontend/src/styles/base/_utilities.scss:133]: related to

Comment (alexbowen) [reply app/frontend/src/styles/base/_utilities.scss:133]: lol, I obviously got halfway through a thought!

**Review by alexbowen — APPROVED** [review]

**Pull request merged**